### PR TITLE
[final] bump version to 3.0.2

### PR DESCRIPTION
**Pull request by aboedo**
bumped the version number to 3.0.2 using fastlane

### CODE REVIEW

File: PurchasesTests/Info.plist [hunk 1]
```diff
@@ -17,7 +17,7 @@
 	<key>CFBundlePackageType</key>
 	<string>BNDL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0</string>
+	<string>3.0.2</string>
```
Comment (aboedo) [inline PurchasesTests/Info.plist]: fastlane uses apple's agvtool under the hood, and it auto-bumps the tests' info.plist. I don't think it's necessarily a bad thing. Thoughts?

**Comment by vegaro** [comment]
It's missing the bum in `RCPurchases.m` right?

File: PurchasesTests/Info.plist (hunk 1 above)
Comment (vegaro) [reply PurchasesTests/Info.plist]: I think it's fine :)

**Comment by aboedo** [comment]
right, sorry about that - I missed it when rolling this back to separate the fastlane changes into a new PR. Adding

**Pull request merged**